evennia/web/static/webclient/js/plugins/goldenlayout.js

```diff
@@ -584,12 +587,33 @@ let goldenlayout = (function () {
     }
 
 
-    //
+    // Listener for realtime changes to the layout name input field.
+    // If the layout name is "default", the save button is disabled
+    // to prevent the perception of overwriting the default layout.
+    $(document).on("input", "#layoutName", function () {
+        console.log("Input changed to:", $(this).val());
```

Comment:
This seems like a leftover debug message (no need to log every change like this, right?)

Reply:
Bingo, thanks for catching that @Griatch - that was just for debugging! Thanks for your eagle eyes!